src/components/PageTitleTag.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import {theme } from 'antd';
+
+const { useToken } = theme;
+
+interface Props {
+  mainText: string;
+  subText?: string;
+}
+const PageTitleCards: React.FC<Props> = ({ mainText, subText }) => {
+  const { token } = useToken();  // Se obtiene el token en PageTitleCards
```

Comment:
can you translate the comments to english?

Reply:
Yes